parachain/node/src/chain_specs/litentry.rs

```diff
@@ -108,8 +108,8 @@ pub fn get_chain_spec_staging() -> ChainSpec {
 pub fn get_chain_spec_prod() -> ChainSpec {
 	get_chain_spec_from_genesis_info(
 		include_bytes!("../../res/genesis_info/litentry.json"),
-		"Litentry",
-		"litentry",
+		"Heima",
+		"heima",
```

Comment:
This won't be compatible with generated JSON file, we have two options here:

1. we keep the `id` unchanged (still as `litentry-*`), minimal changes required - also for collators, but `is_litentry()` might be a bit confusing
2. we change the `id` to `heima-*`, then we'll need to rename the `is_litentry()` fn as well as change the database path